stacker/providers/aws/default.py

```diff
@@ -106,105 +112,65 @@ def cloudformation(self):
 
         return self._cloudformation
 
-    def get_stack(self, stack_name, **kwargs):
-        try:
-            return retry_on_throttling(
-                self.cloudformation.describe_stacks,
-                kwargs=dict(StackName=stack_name))['Stacks'][0]
-        except botocore.exceptions.ClientError as e:
-            if "does not exist" not in e.message:
-                raise
-            raise exceptions.StackDoesNotExist(stack_name)
+    @property
+    def listener(self, existing_topic_arn=None):
+        # deals w/ multiprocessing issues w/ sharing ssl conns
+        # see https://github.com/remind101/stacker/issues/196
+        pid = os.getpid()
+        if pid != self._pid or not self._listener:
+            session = get_session(self.region)
+            self._listener = CloudListener(
+                LISTENER_NAME,
+                session=session,
+                existing_topic_arn=existing_topic_arn
+            )
+            self._listener.start()
+
+        return self._listener
+
+    def set_listener_topic_arn(self, existing_topic_arn):
```

Comment:
So some weirdness here - if `self.listener` has already been called, then calling this will never work, since you test if `self._listener` exists before allowing any changes.